lib/ex_bitmex/rest/orders.ex

```diff
@@ -67,6 +67,20 @@ defmodule ExBitmex.Rest.Orders do
     |> parse_response
   end
 
+  @spec get(credentials, params) ::
+          {:ok, order, rate_limit} | {:error, create_error_reason, rate_limit | nil}
```

Comment:
Wondering that if this get function return just an `order` or `list of orders` ?

Reply:
Later on we can we can add dialyzer to CI to fix this issue I think

Reply:
nice catch. It always will return a list